### ISSUE TITLE
can't login

### ISSUE THREAD

**Issue by justinswabey** [skodaconnect/homeassistant-skodaconnect#257]
**Describe the bug**
On entering email and password on first screen, I get "Could not log in to Skoda Connect. Please check your credentials and verify that the service is working"

**Is this working in MySkoda app?***
Yes, I have logged in to mySkoda app and Skoda Connect website with the same credentials just after getting this message

**Latest working release**
Latest release

**Debug logs**
If applicable, add debug logs. See [README](https://github.com/lendy007/homeassistant-skodaconnect#enable-debug-logging) on how to enable debug logs with response debugging.

**Installation:**
 Core 2024.1.5
Supervisor 2023.12.1
Operating System 11.4
Frontend 20240104.0

**Additional context**

**Comment by dvx76** [comment]
Can you also try logging in with the 'MySkoda Essentials' (not the newer 'MySkoda' app? This one: https://play.google.com/store/apps/details?id=cz.skodaauto.connect&hl=en&gl=US&pli=1 / https://apps.apple.com/gb/app/my%C5%A1koda-essentials/id1161648654

If it still doesn't work please attach HA logs with debug enabled for the integration for the period of time where you get the error in the UI.

**Comment by jhron** [comment]
I had the same issue. Then I tried to install the old application which was superseded by the new version of MyŠkoda. There I tried to log in. It asked me for several confirmations - do you want to allow this, do you want to allow that. After I went through the process, I tried again to log in using HA integration and it was working.

**Comment by justinswabey** [comment]
Have downloaded and logged in to the Essentials app (it actually looks more useful than the more recent one). I accepted all of the permissions it asked me about, but still the same issue logging in to the integration.

Not sure how I can get the logs as I can't enable debugging before I've actually installed the integration fully. 

In HA I'm going to settings>>devises and services>>integrations>>add integration>>search for "Skoda">> click on "Skoda Connect" icon. A popup appears asking me for my Skoda Connect account information, which is where I'm entering my Email and Password....
